app/javascript/css/colors.scss

```diff
@@ -23,6 +23,10 @@
   --success-tint: hsl(130, 61%, 86%);
   --warning-color: hsl(44, 100%, 45%);
   --warning-tint: hsla(51, 100%, 94%, 1);
+  --danger-shade: hsl(0, 100%, 91%);
```

Comment:
naming nitpick - these are more colors than shades, right? at least I think "shade" in the context of the other status colors we use for banners is used to color the text. Probably makes more sense to call these "colors". And to emphasize the difference that these are similar but different than our other status colors, calling them something like `danger-color-2" etc. is probably better IMO

Reply:
https://github.com/ideacrew/enroll/pull/3884/commits/ce3271eb9ccf51b4f4b19769b1b2518291e1ba14